elements/pl-hidden-hints/pl-hidden-hints.py

```diff
@@ -0,0 +1,77 @@
+import chevron
+import lxml
+import prairielearn as pl
+from typing import TypedDict, Tuple, List
+
+
+class HintsDict(TypedDict):
+    """A class with type signatures for the partial scores dict"""
+    html_hints: List[Tuple[int, str]]
```

Comment:
Can you confirm that this should all work with Python 3.8+? I remember in the past we ran into issues adding Python typings, but I don't recall what version we had deployed in production at the time. Presumably pyright would yell at us if we use things that aren't available in the version declared in `pyrightconfig.json`?

Reply:
Pyright will indeed yell at you for using 3.10 constructs since the version is locked at 3.8 in the config. Will double check, but the code we deploy from our repo uses type hints in the same style and we haven't had any issues.

Reply:
Just to double check, copied it over to our course repo and our tooling had no issues. Should be good to go.